NEWS.md

```diff
@@ -11,6 +11,10 @@
 * Support for trailing comma in arrow function parameters
   ([#480](https://github.com/mooz/js2-mode/issues/480)).
 * Support for `for await of` syntax ([#543](https://github.com/mooz/js2-mode/pull/543)).
+* Imenu support for mocha-like (includes Jasmine and Cypress) test
```

Comment:
Please move the entry to the top: they all go in reverse-chronological order here.

Reply:
fixed